pandas/core/frame.py

```diff
@@ -2375,34 +2375,44 @@ def query(self, expr, inplace=False, **kwargs):
             return new_data
 
     def eval(self, expr, inplace=False, **kwargs):
-        """Evaluate an expression in the context of the calling DataFrame
-        instance.
+        """
+        Evaluate expression in the context of the calling DataFrame instance.
+
+        Evaluates a string describing operations on DataFrame columns (only,
+        not specific rows or elements).  This allows `eval` to run arbitrary
```

Comment:
You can move the last sentence to the Notes section, and slightyl expand on 'sanitize' inputs.